.circleci/config.yml

```diff
@@ -359,6 +360,7 @@ jobs:
       name: windows-gpu
     environment:
       CUDA_VERSION: "10.1"
+      PYTHON_VERSION: << parameters.python_version >>
```

Comment:
is this used?

Reply:
it will be used when I fix windows-cuda builds